sparse_ho/criterion/cross_val.py

```diff
@@ -124,7 +125,9 @@ def get_val_grad(
         return val, grad
 
     def get_val_outer(cls, *args, **kwargs):
-        """Get value of outer criterion. TODO
+        """Get value of outer criterion.
+        This is not implemented because for CV the loss is computed on each
+        fold.
 
```

Comment:
```suggestion
        """Get value of outer criterion.

        This is not implemented because for CV the loss is computed on each
        fold.
```

Reply:
done